tribits/core/package_arch/TribitsReadAllProjectDepsFilesCreateDepsGraph.cmake

```diff
@@ -39,7 +39,7 @@
 
 
 # Standard TriBITS system includes
-include(TribitsConstants)
+include("${CMAKE_CURRENT_LIST_DIR}/../common/TribitsConstants.cmake")
```

Comment:
Yes, it makes it so you can use modules out of TriBITS without having to append CMAKE_PREFIX_PATH.  It will allow future refactorings without needing to mess with CMAKE_PREFIX_PATH.  I know this is more verbose but I think it will make the dependencies more explicit and will make maintenance easier overall.

Reply:
Because, initially, I just did as many conversions as needed to support the creation of the tribits/core/test_support/ subdir and moving the test support modules there.   A follow-on PR can convert the rest of the includes (and I would like to use a script for as much of refactoring as possible).